pkg/operator/ceph/csi/spec.go

```diff
@@ -137,11 +139,11 @@ var (
 var (
 	// image names
 	DefaultCSIPluginImage   = "quay.io/cephcsi/cephcsi:v3.10.2"
-	DefaultRegistrarImage   = "registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.9.1"
-	DefaultProvisionerImage = "registry.k8s.io/sig-storage/csi-provisioner:v3.6.3"
-	DefaultAttacherImage    = "registry.k8s.io/sig-storage/csi-attacher:v4.4.2"
-	DefaultSnapshotterImage = "registry.k8s.io/sig-storage/csi-snapshotter:v6.3.2"
-	DefaultResizerImage     = "registry.k8s.io/sig-storage/csi-resizer:v1.9.2"
+	DefaultRegistrarImage   = "registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.10.0"
+	DefaultProvisionerImage = "registry.k8s.io/sig-storage/csi-provisioner:v4.0.0"
+	DefaultAttacherImage    = "registry.k8s.io/sig-storage/csi-attacher:v4.5.0"
+	DefaultSnapshotterImage = "registry.k8s.io/sig-storage/csi-snapshotter:v7.0.1"
+	DefaultResizerImage     = "registry.k8s.io/sig-storage/csi-resizer:v1.10.0"
```

Comment:
```suggestion
	DefaultRegistrarImage   = "registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.10.0"
	DefaultProvisionerImage = "registry.k8s.io/sig-storage/csi-provisioner:v4.0.0"
	DefaultAttacherImage    = "registry.k8s.io/sig-storage/csi-attacher:v4.5.0"
	DefaultSnapshotterImage = "registry.k8s.io/sig-storage/csi-snapshotter:v7.0.1"
	DefaultResizerImage     = "registry.k8s.io/sig-storage/csi-resizer:v1.10.0"
```

Question: can we explore the option in Dependabot to update these version also? So that we don't have to update manually.